SwanHelp/src/handler.ts

```diff
@@ -0,0 +1,43 @@
+import { URLExt } from '@jupyterlab/coreutils';
+import { ServerConnection } from '@jupyterlab/services';
+
+/**
+ * Calls /api/contents/fetch/ with the url to download,
```

Comment:
We should probably mention that this helper function is part of the Gallery functionality. Also the name of the file (`handler.ts`) is too generic and it should change to something more gallery-related so that by looking at the name one knows what's inside? Perhaps `gallery_download_handler.ps` ?